Lib/test/test_traceback.py

```diff
@@ -457,6 +457,41 @@ def f_with_multiline():
         result_lines = self.get_exception(f_with_multiline)
         self.assertEqual(result_lines, expected_f.splitlines())
 
+    def test_caret_multiline_expression_syntax_error(self):
+        # Make sure an expression spanning multiple lines that has
+        # a syntax error is correctly marked with carets.
+        code = """
```

Comment:
You can use textwrap.dedent as in line 277.

Reply:
Thank you, I have amended the code to use textwrap.dedent.